bosh/releases/pre_render_scripts/api/cloud_controller_ng/jobs/patch_file_server_ctl.sh

```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/diego/file_server/templates/file_server_ctl.erb"
```

Comment:
Same.

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d